### PR TITLE
Credit banner vue 876

### PR DESCRIPTION
**Pull request by roger-in-kiva**
PromoCreditBanner component was created in order to have one component for BonusBanner and LendingRewardsBanner.

A story for SB was created to test it.

### CODE REVIEW

File: src/components/WwwFrame/PromotionalBanner/Banners/PromoCreditBanner.vue [hunk 1]
```diff
@@ -0,0 +1,177 @@
+<template>
+	<div
+		v-if="lendingRewardOffered"
+		class="tw-bg-brand tw-text-white md:tw-py-1.5 md:tw-px-0"
+		v-kv-track-event="['TopNav','click-Promo','Lending Reward Banner']"
+	>
+		Make a Kiva loan <br class="tw-inline">and receive a $25 free credit to lend again.
+	</div>
+	<div v-else class="bonus-banner-holder tw-bg-brand tw-text-center tw-py-1.5 tw-px-5">
```
Comment (emuvente) [inline src/components/WwwFrame/PromotionalBanner/Banners/PromoCreditBanner.vue:9]: This should only show if the bonus balance is greater than 0, `v-else-if="bonusBalance > 0"`. I think that will mean that you have to add a third element to this component for the `v-else` condition, `<div v-else></div>`, or use v-show instead.

Comment (roger-in-kiva) [reply src/components/WwwFrame/PromotionalBanner/Banners/PromoCreditBanner.vue]: Thanks for that.

Original condition is this

               `currentActivePromo() {
			if (this.lendingRewardOffered) {
				return LendingRewardsBanner;
			}
			if (this.bonusBalance > 0) { // TODO: skip if on a checkout/basket page
				return BonusBanner;
			}
			return null;
		},`
		
So, I think is enough with `v-else-if="bonusBalance > 0".`

**Review by mcstover — APPROVED** [review]
I think this is good. Is there follow on work to use this in the UI app code to replace the existing banner and remove the swapping between small and large versions?

**Review by ryan-ludwig — APPROVED** [review]
Nice work!

**Comment by ryan-ludwig** [comment]
> I think this is good. Is there follow on work to use this in the UI app code to replace the existing banner and remove the swapping between small and large versions?

Yes, I'll add this into the new Header work.

**Comment by ryan-ludwig** [comment]
Since this replaces several components, can you delete those please and add to this PR.

**Review by emuvente — APPROVED** [review]
Great!

**Comment by ryan-ludwig** [comment]
Thanks for removing the old banners @roger-in-kiva . I think this looks good. @mcstover  can you take a quick look please?

**Review by mcstover — APPROVED** [review]
👏

**Pull request merged**